rllib/BUILD

```diff
@@ -671,6 +671,17 @@ py_test(
 )
 
 # PPO
+
+py_test(
```

Comment:
We shouldn't run this test here (in the CI) at all. It would take too long to complete and requires a GPU, which we don't have in this part of the CI.

Let's just remove this section here and only keep the "auto" fix in the `tuned_examples` file.

Reply:
@sven1977 Thanks for the quick advice!

Reply:
Done